apps/app/src/views/components/TreatmentOptionSearch.tsx

```diff
@@ -0,0 +1,168 @@
+import {
+  Box,
+  ChakraProvider,
+  InputGroup,
+  InputRightElement,
+  Spinner,
+  VStack,
+  Text
+} from '@chakra-ui/react';
+
+import { OptionProps } from 'chakra-react-select';
+import { useEffect, useState } from 'react';
+import { gql, useApolloClient } from '@apollo/client';
+import { usePhoton } from '@photonhealth/react';
+import { useDebounce } from 'use-debounce';
+import { SelectField } from './SelectField';
+import { getMatchingPartsFromSubstring } from '../../utils';
+
+const SearchTreatmentOptionsQuery = gql`
+  query SearchTreatmentOptions($searchTerm: String!) {
+    treatmentOptions(searchTerm: $searchTerm) {
+      id: medicationId
+      form
+      name
```

Comment:
We aren't using all of this info that's being fetched

Reply:
oho snap good point